src/http/websocket_http_client.zig

```diff
@@ -239,7 +239,9 @@ pub fn NewHTTPUpgradeClient(comptime ssl: bool) type {
         pub usingnamespace bun.New(@This());
 
         const HTTPClient = @This();
-
+        fn isDeadPtr(ptr: *HTTPClient) bool {
```

Comment:
This is not a good idea.

Reply:
I think we are only hiting the isDead condition before the socket fixes, will test without this changes

Reply:
If you want to check if the pointer is dead it should be done before it becomes an HTTPClient